test/OpenTelemetry.Instrumentation.Process.Tests/ProcessMetricsTests.cs

```diff
@@ -13,9 +13,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+using System.Collections.Generic;
+using System.Linq;
+using OpenTelemetry.Metrics;
+using Xunit;
 
 namespace OpenTelemetry.Instrumentation.Process.Tests;
 
 public class ProcessMetricsTests
 {
+    private const int MaxTimeToAllowForFlush = 10000;
+
+    [Fact]
+    public void ProcessMetricsAreCaptured()
+    {
+        var exportedItems = new List<Metric>();
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddProcessInstrumentation()
+            .AddInMemoryExporter(exportedItems)
+            .Build();
+
+        meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+
+        Assert.True(exportedItems.Count == 2);
+        Assert.StartsWith("process", exportedItems[0].Name);
```

Comment:
why testing just startswith? the actual name is already covered by tests below.

Reply:
I've removed this assertion.